examples/jsm/libs/lottie_canvas.module.js

```diff
@@ -1,6 +1,14 @@
 const lottie = {};
 
-if (typeof document !== 'undefined') {
+// Checks for SSR including partial DOM implementation and emulation.
+// In these environments, Lottie should not be loaded or it can crash.
+// https://github.com/pmndrs/three-stdlib/pull/398#issuecomment-2600778156
+const isSSR =
```

Comment:
Why isn't this something apps can't check _before_ importing lottie loader? Something like:
```js
if ( isSSR === false ) {

    const { LottieLoader } = await import( 'three/addons/loaders/LottieLoader.js' );

}
```

Reply:
This is because `three/addons` brings in `LottieLoader` when evaluated on the server side.

I can add a CI step for this, which is as simple as `node examples/jsm/Addons.js`.